service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
This is copied from somewhere?

Reply:
this is copied from https://github.com/DataBiosphere/terra-cli/blob/main/notebooks/post-startup.sh. I just want the script to be close to the call site so it's easy to find.